gapic/templates/examples/feature_fragments.j2

```diff
@@ -25,44 +25,78 @@ There is a little, but not enough for it to be important because
 
 {# response handling macros #}
 
+{% macro licenseSection(fileHeader, sample, callingForm) %}
+# -*- coding: utf-8 -*-
+#
+{% for line in fileHeader["copyrightLines"].split("\n") -%}
```

Comment:
I do not really see why this is not just template text.

Reply:
No really good reason, just unit testing and peace of mind.